src/components/search-columns.js

```diff
@@ -33,7 +20,7 @@ function renderCheckbox(column, query, onCheckChange) {
       <Option value={'all'}>ALL</Option>
```

Comment:
This one should be "All" and not "ALL". Thanks!

Reply:
@tvarsis I have updated the branch. do i need to add a morepull request for that.

Thanks